synapse/cortex.py

```diff
@@ -2077,6 +2077,14 @@ async def _addStormPkg(self, pkgdef):
         await self.loadStormPkg(pkgdef)
         await self.pkghive.set(name, pkgdef)
 
+        gates = []
+        perms = []
+        pkgperms = pkgdef.get('perms')
+        if pkgperms:
+            gates = [p['gate'] for p in pkgperms if p.get('gate') is not None]
+            perms = [p['perm'] for p in pkgperms if p.get('perm') is not None]
+        await self.feedBeholder('pkg:add', pkgdef, gates=gates, perms=perms)
```

Comment:
do we want to feed the entire storm package to the consumer? or just the name / version ?

Reply:
There's actually a decent amount of the pkgdef we'd want to feed to the consumer. Optic would want name/desc/logo/version/author/minver/url/iden/optic info from  the def. And we'll probably eventually want the workflow defs as well so we can just automatically load them without roundtripping back to the cortex, so I just left all that in.

I was on the fence about whether or not to shove all the storm code into the message as well. But it wasn't particularly harmful, so I left it.